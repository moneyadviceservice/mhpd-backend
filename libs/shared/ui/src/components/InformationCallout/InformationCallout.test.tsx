import React from 'react';
import { render, screen } from '@testing-library/react';
import { InformationCallout } from '.';

describe('InformationCallout component', () => {
  it('renders correctly', () => {
    render(
      <InformationCallout>
        <div>Information callout</div>
      </InformationCallout>,
    );
    const callout = screen.getByTestId('information-callout');
    expect(callout).toMatchSnapshot();
  });

  it('renders correctly with shadow', () => {
    render(
      <InformationCallout variant="withShadow">
        <div>Information callout</div>
      </InformationCallout>,
    );
    const callout = screen.getByTestId('information-callout');
    expect(callout).toMatchSnapshot();
  });
});
