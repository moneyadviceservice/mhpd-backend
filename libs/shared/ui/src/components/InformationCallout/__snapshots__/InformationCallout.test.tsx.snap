// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InformationCallout component renders correctly 1`] = `
<div
  class="t-information-callout border border-slate-400 rounded-bl-3xl"
  data-testid="information-callout"
>
  <div>
    Information callout
  </div>
</div>
`;

exports[`InformationCallout component renders correctly with shadow 1`] = `
<div
  class="t-information-callout border border-slate-400 rounded-bl-3xl shadow-1md"
  data-testid="information-callout"
>
  <div>
    Information callout
  </div>
</div>
`;
